rotkehlchen/chain/ethereum/modules/convex/decoder.py

```diff
@@ -48,6 +47,7 @@ class ConvexDecoder(DecoderInterface):
     def _decode_convex_events(self, context: DecoderContext) -> DecodingOutput:
         amount_raw = hex_or_bytes_to_int(context.tx_log.data[0:32])
         interacted_address = hex_or_bytes_to_address(context.tx_log.topics[1])
+        found_relevant_event = False
```

Comment:
add a comment or lets use a better name. What about `found_event_modifying_balances`

Reply:
okay, will change the name